src/components/DetailSlides.js

```diff
@@ -18,14 +21,29 @@ export default function DetailSlides({ images }) {
     setSlideNumber(index);
   };
 
+  const handleTouchStart = (event) => {
+    console.log(event);
+    setTouchStartX(event.changedTouches[0].clientX);
+  };
+
+  const handleTouchEnd = (event) => {
+    if (touchStartX - event.changedTouches[0].clientX > 30) {
+      rightButton.current.click();
+    } else if (event.changedTouches[0].clientX - touchStartX > 30) {
+      leftButton.current.click();
+    }
+  };
```

Comment:
좌표값을 구하면 간단한 방법이었군요! 👍
```suggestion
  const handleTouchEnd = (event) => {
    if (touchStartX - event.changedTouches[0].clientX > 30) {
      handleSlideMove(-1)
    } else if (event.changedTouches[0].clientX - touchStartX > 30) {
      handleSlideMove(1)
    }
  };
```

다만 이미 handleSlideMove라는 슬라이드 움직이는 로직을 구현했기 때문에 ref로 돔을 직접 조작하지 않고 다음과 같이 설정하는게 좋을 것 같습니다.

Reply:
아 그러네요 코드 리뷰하면서 코드 이해 충분히 했다 생각했는데
꼼꼼하지 못한 부분이네요 시도해보겠습니다